doc/spec.md

```diff
@@ -3025,6 +3025,11 @@ application-specific dialect) without breaking existing programs.
 
 `True` and `False` are the two values of type `bool`.
 
+### abs
+
+`abs(x)` returns the absolute value of the same type.
```

Comment:
"not a real number" is not correct: the function succeeds for +/-inf and nan, none of which are real.

Say:

"abs(x) returns the absolute value of its argument x, which must be an int or float. 
The result has the same type as x."